lib/data/hive/hive_client_name_box.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:hive_flutter/hive_flutter.dart';
+import 'package:matrix/matrix.dart';
+
+class HiveClientNameBox {
```

Comment:
Should have an abstract for other box in the future

Reply:
i dont think it's necessary because these methods (`insertClientName`, ...) in only for this specific box. For abstract class, we already have `CollectionBox`